subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -916,13 +916,13 @@ struct pdu_adv_aux_ptr *ull_adv_aux_lll_offset_fill(uint32_t ticks_offset,
 {
 	struct pdu_adv_com_ext_adv *pri_com_hdr;
 	struct pdu_adv_aux_ptr *aux;
-	struct pdu_adv_hdr *h;
+	struct pdu_adv_ext_hdr *h;
 	uint32_t offs;
 	uint8_t *ptr;
 
 	pri_com_hdr = (void *)&pdu->adv_ext_ind;
-	h = (void *)pri_com_hdr->ext_hdr_adi_adv_data;
-	ptr = (uint8_t *)h + sizeof(*h);
+	h = (void *)pri_com_hdr->ext_hdr_adv_data;
+	ptr = h->data;
```

Comment:
There is some changes to the implementation here, hence blocking this PR until #29778 is merged. Otherwise changes proposed here looks good.